microsoft-edge/extensions-chromium/developer-guide/sidebar.md

```diff
@@ -5,13 +5,11 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 06/27/2023
+ms.date: 08/18/2023
 ---
 # Extensions in the Microsoft Edge sidebar
 
-<!-- compare https://developer.chrome.com/docs/extensions/reference/sidePanel/ -->
-
-This is a forthcoming feature; sidebar extensions are not yet in a stable release of Microsoft Edge.
+The sidebar extensions feature is being rolled out to a growing user base for all channels of Microsoft Edge.<!-- todo: delete sentence -->
```

Comment:
I recommend moving this para down to make it the 2nd para in the article. 

Also adding "Note: " would make it clear that this is an important piece of content to read.